### PR TITLE
docs(integrity_checking): improve clarity

### PR DESCRIPTION
**Pull request by ca-d**
I'm making this proposal for improving clarity of the language because I personally found the original phrasing a bit confusing in some places.

### CODE REVIEW

File: basics/modules/integrity_checking.md [hunk 1]
```diff
@@ -16,16 +16,15 @@ integrity checking and lock files.
 Deno can store and check subresource integrity for modules using a small JSON
 file. To opt into a lock file, either:
 
-1. Create a `deno.json` file in the current or an ancestor directory, which will
-   automatically create an additive lockfile at `deno.lock`.
```
Comment (kt3k) [inline basics/modules/integrity_checking.md:20]: I think the word 'additive lockfile' is intentional and it tries to describe that the contents of it are sum of all dependencies of all relevant files (correct me if I'm wrong cc @dsherret @bartlomieju )

Comment (ca-d) [reply basics/modules/integrity_checking.md:20]: Thanks for the clarification, me not understanding this point was actually the chief impulse for this suggestion. If that is the case, I propose to either add an explanation of "additive lockfile" or maybe rename it to "cumulative lockfile", if that more closely reflects the intended meaning.

Comment (kt3k) [reply basics/modules/integrity_checking.md:20]: @dsherret What do you think about the above suggestion?

Comment (dsherret) [reply basics/modules/integrity_checking.md:20]: Maybe let's just remove the word "additive" and "additional" here.

**Pull request closed without merging**